source/cli/realm-cli-apps-init.txt

```diff
@@ -1,4 +1,4 @@
-.. _realm-cli-apps-init:
+.. _realm-cli-Apps-init:
```

Comment:
nop

```suggestion
.. _realm-cli-apps-init:
```

also this will fix a snooty build error

Reply:
thank you